rust/main/agents/relayer/src/relayer.rs

```diff
@@ -216,63 +216,53 @@ impl BaseAgent for Relayer {
         let contract_sync_metrics = Arc::new(ContractSyncMetrics::new(&core_metrics));
 
         start_entity_init = Instant::now();
-        let message_syncs: HashMap<_, Arc<dyn ContractSyncer<HyperlaneMessage>>> = settings
-            .contract_syncs::<HyperlaneMessage, _>(
-                settings.origin_chains.iter(),
-                &core_metrics,
-                &contract_sync_metrics,
-                dbs.iter()
-                    .map(|(d, db)| (d.clone(), Arc::new(db.clone())))
-                    .collect(),
-                false,
-                settings.tx_id_indexing_enabled,
-            )
-            .await?
-            .into_iter()
-            .map(|(k, v)| (k, v as _))
+
+        let advanced_log_meta = false;
```

Comment:
It is good idea to extract advance log meta flag into a variable! It will be better if it is just a constant in Relayer.